schema/article.js

```diff
@@ -18,9 +18,18 @@ const ArticleType = new GraphQLObjectType({
     title: {
       type: GraphQLString,
     },
+    published_at: {
+      type: GraphQLString,
+    },
+    updated_at: {
+      type: GraphQLString,
+    },
```

Comment:
Can you use the date field here? https://github.com/1aurabrown/metaphysics/blob/72fb060615a55f13be5469729e0edb98405a3c7f/schema/partner_show.js#L90